### ISSUE TITLE
How do you handle different intrinsic parameters for 3D detection?

### ISSUE THREAD

**Issue by XinnWang** [xingyizhou/CenterTrack#174]
Hi,

For 3D detection, distance is calculated by depth and intrinsic parameters of cameras. I noticed that in nuScenes dataset, all 6 cameras have different intrinsic parameters. Since you are using only one model, how do you handle different intrinsic parameters for 3D detection? Thanks.

**Comment by nmll** [comment]
yes, I also have the question, How does this work convert camera 2Dboundingbox to 3D in nuscenes?
